### ISSUE TITLE
Linting wrong line

### ISSUE THREAD

**Issue by artemartemov** [attenzione/SublimeLinter-scss-lint#39]
Hey there, I have the linter working perfectly on my machine, but when I was installing for a coworker it seems that it is linting incorrect lines and wrong errors are reading out. What could cause this? The setup was the same as mine and the config file is also the same and it's within the users/username directory. 

How could I debug this?

**Comment by RiaanWest** [comment]
I'm experiencing the same problem. The linting feedback is correct, but the wrong lines are highlighted.

**Comment by ahallicks** [comment]
I have just started getting this issue too. I thought it might be related to my theme (Material) that just updated, but not sure.

After checking the console I can see that the correct line is highlighted in the output from the linter, but it doesn't appear to be applying this to the gutter styles at all.

Weirdly there also seems to be no pattern to where the errors now show up. At first mine all showed up on the first line of the file. Then I had errors showing up on line 3 and warnings on line 32! A colleague currently has warnings on his first 5 lines, none of which correspond to the given warnings.

It's an odd one!

**Comment by Macke105** [comment]
These problems started in v0.49.0, reverting back to 0.48.0 solved my problems.

**Comment by alexmccabe** [comment]
@Macke105 v0.48.0 of what? This issue just started for me and I don't have Material Theme installed.

**Comment by antonhedstrom** [comment]
Seems that the format of the output from scss_lint changed from 0.48 to 0.49. They have now added not only which line the error is at, but also which column.

**Comment by ahallicks** [comment]
@antonhedstrom that's a good point! Looking at the console for the output of the linter I can see some fairly major differences between the format of JSHist and Scss-lint (when I presume they are supposed to be the same.

Scss-lint:
```
SublimeLinter: scss output:
C:\path\to\file\_file.scss:353:1 [E] TrailingSemicolon: Declaration should be terminated by a semicolon 
```

JSHint:
```
SublimeLinter: jshint output:
C:\path\to\file\file.js: line 177, col 16, Missing semicolon. (W033)
```
